### PR TITLE
[stable/elasticsearch-curator] Update to 5.7.6 by changing docker repository

### PR DESCRIPTION
**Pull request by desaintmartin**
Update to 5.7.6 by changing docker repository.
Also add desaintmartin to maintainers in order to help.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/elasticsearch-curator/README.md [hunk 1]
```diff
@@ -23,6 +23,17 @@ To install the chart, use the following:
...
+## Upgrading an existing Release to a new major version
+
+A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an
+incompatible breaking change needing manual actions.
+
+### To 2.0.0
+
+v2.0.0 uses docker image from `elasticsearch-curator` author, which differs in its way to install curator.
+
+If you have an hardcoded `command` value, please update it to follow the new `curator` executable path: `/curator/curator` (which is not in PATH).
```
Comment (carrodher) [inline stable/elasticsearch-curator/README.md]:
```suggestion
If you have a hardcoded `command` value, please update it to follow the new `curator` executable path: `/curator/curator` (which is not in PATH).
```

**Comment by carrodher** [comment]
LGTM, although as I am not an owner I can't add the proper labels

**Comment by desaintmartin** [comment]
@tmestdagh, are you still interested into maintaining this chart? If so, we need to get you added into the helm org, otherwise, we need to get your name removed from the list.

**Comment by desaintmartin** [comment]
@tmestdagh I removed you from list in order to continue but feel free to comment here or do a PR to add you back.

**Review by cpanato — APPROVED** [review]
/lgtm

**Pull request merged**